rand_os/src/lib.rs

```diff
@@ -362,3 +382,58 @@ compile_error!("enable either wasm_bindgen or stdweb feature");
     target_env = "sgx"
 )))]
 compile_error!("OS RNG support is not available for this platform");
+
+// Due to rustwasm/wasm-bindgen#201 this can't be defined in the inner os
+// modules, so hack around it for now and place it at the root.
+#[cfg(all(feature = "wasm-bindgen", target_arch = "wasm32"))]
+#[doc(hidden)]
+#[allow(missing_debug_implementations)]
+pub mod __wbg_shims {
```

Comment:
Does this module have to be public? Can't it be `pub(crate)`?

Reply:
I don't know. The module isn't required at all for building and we don't have a running test-suite.

Reply:
Either way I don't think it should be public, and if for some reason it has to be, then at least we should hide it from docs.